### ISSUE TITLE
Epoch time does not show elapsed time for more than 59 seconds.

### ISSUE THREAD

**Issue by shrit** [mlpack/ensmallen#244]
Hello everyone,

I have just noticed a small issue in the estimated time callback. I am training a neural network on a large dataset, the training time for each epoch is taking more than one minute, (several minutes per epoch). 
My understanding is that at the end of each epoch the ETA callback prints the elapsed training time for each epoche. If this is the case, the printed time does not show the minutes elapsed but only shows the seconds and not the seconds after `59`. In the following, each epoch took at least 3 to 4 minutes.
```
9563.88
Epoch 1
5819/5819 [====================================================================================================] 100% - 18s 34ms/step - loss: 40.7373
Validation loss: 770.42
Epoch 2
5819/5819 [====================================================================================================] 100% - 10s 43ms/step - loss: 34.7851
Validation loss: 759.774
```
To reproduce the issue, use a large size dataset `> 1M` lines and train it on the CPU. In the end, you should see the elapsed time only in seconds.

**Comment by zoq** [comment]
Thanks for the report, the issue is here:

https://github.com/mlpack/ensmallen/blob/cdfae670da608ef8498dee1e649f79f1d3279ef9/include/ensmallen_bits/callbacks/progress_bar.hpp#L221

I'll fix it.

**Comment by zeph1yr** [comment]
Hello @zoq, Can I submit a pull request, looks like a good first issue. Thanks!

**Comment by zoq** [comment]
> Hello @zoq, Can I submit a pull request, looks like a good first issue. Thanks!

Ahh, yes that would be awesome.

**Comment by conradsnicta** [comment]
Should be solved by #337. Please reopen if it doesn't work properly.
